### PR TITLE
initial scan of hostiles

### PR DESCRIPTION
**Pull request by ccMatrix**
some hostiles I found in the data. some might be incomplete. Do we need a `scout: boolean` field in there?

### CODE REVIEW

**Comment by WebSpider** [comment]
Given the digit parsing, maybe a good idea to reparse this with adjusted regexes for seperators as well

File: hostiles/borg_tactical_probe_29_battleship.json [hunk 1]
```diff
@@ -16,21 +16,21 @@
...
     "defense": {
-      "armor": 23,
+      "armor": 23388,
       "dodge": 5848,
       "shield_deflect": 5848
     },
     "health": {
-      "hull_health": 2714,
-      "shield_health": 2036
+      "hull_health": 2,
```
Comment (WebSpider) [inline hostiles/borg_tactical_probe_29_battleship.json:32]: This doesn't look right to me .. Borg probes dont have a hull health of 2

File: hostiles/exchange_transport_28_survey.json [hunk 2]
```diff
@@ -16,21 +16,21 @@
...
-      "dodge": 13,
-      "shield_deflect": 13
+      "armor": 13625,
+      "dodge": 13625,
+      "shield_deflect": 13625
     },
     "health": {
-      "hull_health": 1210,
-      "shield_health": 1147
+      "hull_health": 1,
```
Comment (WebSpider) [inline hostiles/exchange_transport_28_survey.json:32]: All the hull healths seem to be reduced to their int(original_value/1000)

**Review by WebSpider — CHANGES_REQUESTED** [review]

**Comment by ccMatrix** [comment]
let's close this and move it over to the full reparse PR

**Pull request closed without merging**